openapi/paths/api@register@{provider}.yaml

```diff
@@ -10,17 +10,16 @@ post:
       description: |
         The type of device that is connecting to the service.
 
-        As of this version, this is expected to always be "Digistorm". Apple
-        (APNS) or Google Android (GCM) are still supported, but deprecated: the
-        Digistorm provider will handle the sending of notifications via APNS or
-        GCM internally.
+        As of this version, this is expected to always be "Firebase". Apple
```

Comment:
I think we can remove any expectations at this point, we have a completely agnostic delivery system that allows us to support any number of notification delivery systems.  So apart from saying APNS and GCM are deprecated.  Both digistorm and firebase are fully supported.